include/mbedtls/x509_crt.h

```diff
@@ -347,7 +387,26 @@ int mbedtls_x509_crt_parse_file( mbedtls_x509_crt *chain, const char *path );
  */
 int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path );
 #endif /* MBEDTLS_FS_IO */
-
+/**
+ * \brief          Parses the subject alternative name list of a given certificate;
+ *
+ * \param crt      The X509 certificate to parse.
+ *
+ * \param san      A list holding the parsed certificate.
+ *
+ * \note           Only "dnsName" and "otherName" of type hardware_module_name,
+ *                 as defined in RFC 4180 is supported.
+ *
+ * \note           Any unsupported san type is ignored.
+ *
+ * \note           The function allocates a list of mbedtls_x509_subject_alternative_name
+ *                 and it is the caller's responsibility to free it.
+ *
+ * \return         Zero for success and negative
+ *                 value for any other failure.
```

Comment:
Question: I never remember the correct way of doing this. Should it be a different `\return` command for every possible value (or group of values)? Or is it fine just having one return like this?

Reply:
You are right, in some headers we do it in separate lines, but here we seem to use a single `\return` command per entry. Therefore I would prefer to leave it like this for local consistency.

Reply:
I think we should stick the desired format, at least when creating new documentation.